src/chart/chart.css

```diff
@@ -19,4 +19,15 @@
 
 .plot-area {
   overflow: hidden;
+}
+
+.y-axis .label,
+.x-axis .label,
+.title .label {
+    dominant-baseline: middle;
+    text-anchor: middle;
+}
+
+.cartesianChart {
```

Comment:
Should we have the rule that all classes are dash-separated rather than camel-cased?